tests/integration/run_tests.rs

```diff
@@ -5177,3 +5178,107 @@ fn run_etag_delete_source_cache() {
       "[WILDCARD]Cache body not found. Trying again without etag.[WILDCARD]",
     );
 }
+
+#[test]
+fn code_cache_test() {
+  let script_dir = TempDir::new();
+  script_dir.write("main.js", "console.log('Hello World - A');");
+  let prg = util::deno_exe_path();
+  let deno_dir = TempDir::new();
+
+  // First run with no prior cache.
+  {
+    let output = Command::new(prg.clone())
+      .env("DENO_DIR", deno_dir.path())
+      .current_dir(util::testdata_path())
+      .arg("run")
+      .arg("-Ldebug")
+      .arg(format!("{}/main.js", script_dir.path()))
+      .output()
+      .expect("Failed to spawn script");
+
+    let str_output = std::str::from_utf8(&output.stdout).unwrap();
+    assert!(str_output.contains("Hello World - A"));
+
+    let debug_output = std::str::from_utf8(&output.stderr).unwrap();
+    // There should be no cache hits yet, and the cache should be created.
+    assert!(!debug_output.contains("V8 code cache hit"));
+    assert!(Regex::new(
+      r"Updating V8 code cache for ES module: file://(.+)/main.js"
+    )
```

Comment:
No need for the regex crate here -- we can use `assert_matches_text` to avoid a new dep here.